src/main/kotlin/org/rust/lang/core/types/regions/Scope.kt

```diff
@@ -205,7 +205,6 @@ private class RegionResolutionVisitor(contextOwner: RsInferenceContextOwner) : R
             }
             stmt.accept(this)
         }
-        block.expr?.let { visitExpr(it) }
```

Comment:
Just in case, shouldn't we process `expandedTailExpr` here instead?

Reply:
Hmm, yes, it looks like that. Couldn't you make a test for this?